hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -321,6 +321,7 @@ public static boolean isReadOnly(
     case TenantAssignAdmin:
     case TenantRevokeAdmin:
     case SetRangerServiceVersion:
+    case RecoverLease:
```

Comment:
Ozone do not have any acquireLease kind of interface, this name will be confusing wrt ozone. It can be RecoverOpenKey.